### DISCUSSION TITLE
A new way to check characters

### DISCUSSION THREAD

**Discussion by Neiroromo** [Andrewthe13th/Inventory_Kamera#409] [Ideas or Feature requests]
Since hoyoverse allows you to call some characters by custom names, it will be more logical to identify the characters by their constellation
![image](https://user-images.githubusercontent.com/70837136/207250478-151a9fa4-7bd3-4f6d-88f5-20fc3471e604.png)
![image](https://user-images.githubusercontent.com/70837136/207250548-4c9095d4-bab7-467b-8636-052af4333e44.png)

**Comment by Cupcak3** [comment]
That's for showing me this. While the current logic *can* support adding constellations to identify characters, it would mean going back and manually adding it to each character as well as updating code for it. 

But I agree that this can uniquely identify characters better than currently and will probably be included once I begin working on a new auto updater
